leakcanary-android-leak-deobfuscation-plugin/src/main/java/com/squareup/leakcanary/leak/deobfuscation/plugin/LeakCanaryDeobfuscationExtension.kt

```diff
@@ -0,0 +1,7 @@
+package com.squareup.leakcanary.leak.deobfuscation.plugin
+
+import com.android.build.gradle.api.BaseVariant
+
+open class LeakCanaryDeobfuscationExtension {
+  var filterObfuscatedVariants: (BaseVariant) -> Boolean = { false }
```

Comment:
The default could be `variant.defaultType.minifyEnabled`: https://static.javadoc.io/com.android.tools.build/builder/1.2.0/com/android/builder/core/DefaultBuildType.html#isMinifyEnabled()

Reply:
The kdoc should talk about the default value.